### ISSUE TITLE
Add ability to not commit when on certain branches

### ISSUE THREAD

**Issue by ClashTheBunny** [muesli/gitomatic#2]
It would be nice to have a few protected branches passable as command line options.  Something so that I can just leave this running and don't commit to a release branch a bunch accidentally.

**Comment by BenedictRasmussen** [comment]
Perhaps this bastardizes this issue, but I would generalize this a little bit. Personally I commit at certain points (when i get things to a semi-stable point) so that I can roll back at any point. It would be nice to have a toggle of some kind on startup to toggle whether you are monitoring upstream for pulls, monitoring locally for pushes, or both.

**Comment by muesli** [comment]
Yeah, that should probably be a separate ticket. That said, both are nice suggestions, thank you!

**Comment by muesli** [comment]
@BenedictRasmussen Your suggestion has just landed in master.
